homeassistant/components/speedtestdotnet/const.py

```diff
@@ -1,10 +1,28 @@
 """Consts used by Speedtest.net."""
 
 DOMAIN = "speedtestdotnet"
+
+SPEED_TEST_SERVICE = "speedtest"
 DATA_UPDATED = f"{DOMAIN}_data_updated"
 
 SENSOR_TYPES = {
     "ping": ["Ping", "ms"],
     "download": ["Download", "Mbit/s"],
     "upload": ["Upload", "Mbit/s"],
 }
+
+CONF_SERVER_ID = "server_id"
+CONF_MANUAL = "manual"
+
+DEFAULT_NAME = "SpeedTest"
+DEFAULT_SCAN_INTERVAL = 60
```

Comment:
I don't think 60 seconds is a good default scan interval when the previous default was an hour. We should maintain the same default to avoid the breaking change.

Reply:
This is not seconds, it is minutes.

Reply:
Interval every hour is nice 👍